source/atlas-data-lake-testing/tests/estimatedDocumentCount.yml

```diff
@@ -13,9 +13,6 @@ tests:
       -
         command_started_event:
           command:
-            aggregate: *collection_name
-            pipeline:
-              - $collStats: { count: {} }
```

Comment:
If you've not done so already, can you take a quick grep through the specifications repo and ensure that `$collStats` appears no where else? This request assumes any instance of it would have been related to estimatedDocumentCount, so do take that into consideration if there's another legitimate instance left behind.